_config/queuedjobs.yml

```diff
@@ -7,6 +7,7 @@ Injector:
   QueuedJobService:
     properties: 
       queueHandler: %$QueueHandler
+      queueRunner: %$DoormanRunner
```

Comment:
This is a circular dependency and might get messy;

QueuedJobService requires DoormanRunner
DoormanRunner extends BaseRunner
BaseRunner requires QueuedJobService (https://github.com/nyeholt/silverstripe-queuedjobs/blob/tractorcow-pulls/add-additional-core-engine/code/tasks/engines/BaseRunner.php#L10)